pages/settings/activity.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" :offset-md="1" md="5" class="pt-0 pb-4">
+        <!-- Administrator settings -->
+        <v-list
+          v-if="$auth.user.Policy.IsAdministrator"
+          two-line
+          class="mb-4"
+          disabled
+        >
+          <v-list-item-group>
+            <v-list-item v-for="activity in activityList" :key="activity.Id">
+              <v-list-item-avatar>
+                <v-icon>mdi-shield-account-outline</v-icon>
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="activity.Name" />
+                <v-list-item-subtitle v-text="activity.ShortOverview" />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-list-item-subtitle v-text="activity.Date" />
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+        <!-- Mobile links -->
+        <v-list v-if="$vuetify.breakpoint.mobile">
```

Comment:
This isn't needed here :p

Reply:
this was still in initial stage so a lot of extras and wrongs are there :P. I just opened PR to reserve this feature. 
I should have focused more on code to remove all these.